test/__test__/query.test.ts

```diff
@@ -1,52 +1,57 @@
-import * as AWS from "aws-sdk";
+import { DynamoDBClient, DynamoDBClientConfig } from "@aws-sdk/client-dynamodb";
+import { DynamoDBDocumentClient } from "@aws-sdk/lib-dynamodb";
 import { Table } from "../../src/index";
 
-const tableWithSmallData = new Table("Clevo-Processed-Speech-Table");
-const tableWithIndexes = new Table("Clevo-Raw-Speech-Table");
+let tableWithSmallData;
+let tableWithIndexes;
 console.error = jest.fn();
 
 beforeAll(() => {
-  const dynamoOptions = {
-    apiVersion: '2012-08-10',
+  const dynamoOptions: DynamoDBClientConfig = {
     region: "us-west-2",
     endpoint: "http://localhost:8000",
-    accessKeyId: "test",
-    secretAccessKey: "test"
+    credentials: {
+      accessKeyId: "test",
+      secretAccessKey: "test",
+    },
   };
-  const localDynamodb = new AWS.DynamoDB(dynamoOptions);
-  const localClient = new AWS.DynamoDB.DocumentClient(dynamoOptions);
+  const localDbClient = new DynamoDBClient(dynamoOptions);
+  const localDocClient = DynamoDBDocumentClient.from(localDbClient);
 
-  Table.replaceDynamoClient(localDynamodb, localClient);
+  Table.replaceDynamoClient(localDbClient, localDocClient);
```

Comment:
Thanks for the MR! Could you add a couple of tests to test the new optional config param in the constructor?

Namely, something like this
```
tableWithSmallData = new Table("Clevo-Processed-Speech-Table", {region: "ap-northeast-1"});
``` 
And test the region is indeed changed?

Reply:
New tests were added. 
For each operation, 2 new tests were added to test default region and override region, respectively.

Reply:
e.g. 
1. test default region: https://github.com/Eon-Labs/dynamo-light/pull/36/files#diff-90b0fdc25ec525fa2f62184130c1a7e750018144d97abf7171d6aa71ddb20b9bR75
2. test override region: https://github.com/Eon-Labs/dynamo-light/pull/36/files#diff-90b0fdc25ec525fa2f62184130c1a7e750018144d97abf7171d6aa71ddb20b9bR89